src/components/TruncatedList/TruncatedList.stories.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file Truncated list stories.
+ * @copyright IBM Security 2020
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { number, select, color, text } from '@storybook/addon-knobs';
+import { components } from '../../../.storybook';
+import TruncatedList from '.';
+import { createChildrenArray } from './_mocks_';
+
+storiesOf(components(TruncatedList.name), module).add(
+  'default',
+  () => (
+    <TruncatedList
+      className="other-class"
+      as={select('list element (as)', ['ul', 'ol', 'dl'], 'ul')}
+      scrollGradientColor={color(
+        'scroll gradient color (scrollGradientColor)',
+        TruncatedList.defaultProps.scrollGradientColor
+      )}
+      expandLabel={text('Expand button label (expandLabel)', 'View more')}
+      collapseLabel={text('Collapse button label (collapseLabel)', 'View less')}
+      truncateThreshold={number(
+        'Number of items to show before truncation (truncateThreshold)',
+        10
+      )}
+      collapsedItemLimit={number(
+        'Number of items to show when collapsed (collapsedItemLimit)',
+        5
+      )}
+      expandedItemLimit={number(
+        'Number items to show when expanded (expandedItemLimit)',
+        10
+      )}
+    >
+      {createChildrenArray(number('Number of items in list', 15))}
+    </TruncatedList>
+  ),
+  {
+    info: {
+      text: `## Truncated list
+This component is a list that accepts any number of list items (li). It allows consumers to control
+how many items are revealed to the user while giving the user the ability to expand the list and see
+the entire list. There are three props that control how many items a user can see at once:
+
+  * \`truncateThreshold\`: This prop will give consumers control of how many items are rendered
```

Comment:
I love the formatting you did here but unfortunately the rendered `ul`, `li`, etc elements are not styled in storybook: 

<img width="925" alt="Screen Shot 2020-04-07 at 9 18 00 AM" src="https://user-images.githubusercontent.com/9057921/78680200-d2891d80-78b0-11ea-91c9-0d77f54a157f.png">

There are a couple ways to get around this, I think 🤔 One option is to just add some storybook/demo-specific styling, so that the lists are formatted? Or you could not use markdown here and try to apply a `List` component.

Reply:
So um I don't know what happened, but the list styles got all different since I last looked at this. Suddenly all lists now have bullets and padding, which resolved this comment. It broke some of the other components related to the `TruncatedList`, but I fixed them.